src/libs/actions/Report.js

```diff
@@ -897,7 +897,7 @@ function deleteReportComment(reportID, reportAction) {
         lastMessageText: '',
         lastVisibleActionCreated: '',
     };
-    if (reportAction.reportActionID && reportAction.childVisibleActionCount > 0) {
+    if (reportAction.reportActionID && reportAction.childVisibleActionCount === 0) {
```

Comment:
Do we need to check `reportAction.reportActionID` here?

```
    const optimisticReportActions = {
        [reportActionID]: {
``` 

Doesn't this imply it's truthy? If it's still needed, can it be replaced with `if (reportActionID && ...`?

Reply:
That's a good point. If it's not defined then something is wrong because we are about to save an object with a key of `undefined`. I think this was probably just overly cautious programming.

Reply:
If there is a reason it should have been documented better 🤷‍♂️